code/modules/surgery/organs/organ_external.dm

```diff
@@ -46,8 +46,8 @@
 	var/damage_msg = "<span class='warning'>You feel an intense pain</span>"
 	var/broken_description
 
-	var/open = 0  // If the body part has an open incision from surgery
-	var/sabotaged = 0 //If a prosthetic limb is emagged, it will detonate when it fails.
+	var/open = 0  // If the body part has an open incision from surgery. Can have values > 1.
```

Comment:
![20220706-211829-Code](https://user-images.githubusercontent.com/7831163/177645188-7a2f84e5-1430-4210-8ec5-b944be3235ef.png)

Yikes!...

Reply:
Old surgery hoooo. Worst part, it is still sort of used